controller/workitem_blackbox_test.go

```diff
@@ -1288,6 +1288,35 @@ func (s *WorkItem2Suite) TestWI2SuccessCreateWorkItemWithDescriptionAndMarkup()
 	assert.Equal(s.T(), rendering.SystemMarkupMarkdown, wi.Data.Attributes[workitem.SystemDescriptionMarkup])
 }
 
+func (s *WorkItem2Suite) TestWI2SuccessUpdateWorkItemWithDescriptionAndMarkup() {
+	// given
+	c := minimumRequiredCreatePayload()
+	c.Data.Attributes[workitem.SystemTitle] = "Title"
+	c.Data.Attributes[workitem.SystemDescription] = rendering.NewMarkupContent("Description", rendering.SystemMarkupMarkdown)
+	c.Data.Relationships.BaseType = newRelationBaseType(workitem.SystemBug)
+
+	// when
+	_, wi := test.CreateWorkitemsCreated(s.T(), s.svc.Context, s.svc, s.workitemsCtrl, *c.Data.Relationships.Space.Data.ID, &c)
+	assert.Equal(s.T(), "Description", wi.Data.Attributes[workitem.SystemDescription])
+	assert.Equal(s.T(), "<p>Description</p>\n", wi.Data.Attributes[workitem.SystemDescriptionRendered])
+	updatePayload := minimumRequiredUpdatePayload()
+	updatePayload.Data.ID = wi.Data.ID
+	updatePayload.Data.Attributes["version"] = wi.Data.Attributes["version"]
+	content := "```\n" +
+		"{\n" +
+		`	"data": "hello world"\n` +
+		"}\n" +
+		"```"
+	renderedContent := "<pre><code class=\"prettyprint\"><span class=\"pun\">{</span>\n\t<span class=\"str\">&#34;data&#34;</span><span class=\"pun\">:</span> <span class=\"str\">&#34;hello world&#34;</span><span class=\"pun\">\\</span><span class=\"pln\">n</span><span class=\"pun\">}</span>\n</code></pre>\n"
```

Comment:
here you could use single quotes to avoid having to replace the tabs and line returns with `\t` and `\n` respectively. That also makes the code easier to read (although, you need to remove all formatting indentations)

Reply:
Please look at https://github.com/fabric8-services/fabric8-wit/pull/2165/commits/5f2b92330cbd15457022d86d22a9f47deea76649#diff-98914f859911696f3cdd98dd289d00faR1308. 
I've tried to make it more readable. I'd like to know if there is a better way of doing it.